src/index.ts

```diff
@@ -201,9 +198,6 @@ function _decode(input: Buffer): Decoded {
     }
 
     innerRemainder = safeSlice(input, llength, totalLength)
-    if (innerRemainder.length === 0) {
-      throw new Error('invalid rlp, List has a invalid length')
```

Comment:
This one can also be removed but is a bit more complex. We only have `innerRemainder.length === 0` in case that `llength` and `totalLength` are equal (any other combination would throw `safeSlice`. However, we have `totalLength = llength + length` and therefore we have that `length` should be 0, but a few lines above there is a check whether `length < 56`; if that is the case then it throws. So, this condition can also not be reached.